src/main/resources/locale_de_DE.properties

```diff
@@ -240,3 +240,5 @@ words.refMessage=Referenz Nachricht
 words.reputation=Reputation
 words.role=Rolle
 words.unknown=Unbekannt
+command.repSettings.sub.emojidebug.explain.noDonor=Spender hat nichts in diesem Kanal geschrieben
+command.repSettings.sub.emojidebug.explain.prompted=Der Benutzer wurde zur Eingabe der Reputation aufgefordert
```

Comment:
That sounds wrong xD

Reply:
Then suggest something different xD

Reply:
I think that sounds a bit better. Not 100% happy with that, but I think better than the current
```suggestion
command.repSettings.sub.emojidebug.explain.prompted=Der Benutzer wurde nach Reputationsvergabe gefragt
```